lib/generators/open_stax/rescue_from/install/install_generator.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails/generators'
+
+module OpenStax
+  module RescueFrom
+    class InstallGenerator < Rails::Generators::Base
+      source_root File.expand_path("../templates", __FILE__)
+
+      def copy_templates
+        copy_file "any.html.erb", "app/views/errors/any.html.erb"
+      end
+
+      def copy_initializer
+        copy_file "rescue_from.rb", "config/initializers/rescue_from.rb"
+      end
+
+      def configure_environments
+        test_file = "test.rb"
+        test_middleware = <<-TEST
+  config.middleware.use ExceptionNotification::Rack, email: {
+    email_prefix: "[Tutor] (TEST) ",
+    sender_address: %{"OpenStax Tutor" <noreply@openstax.org>},
```

Comment:
Regardless of where this config ends up, we'll want these options, `email_prefix`, `sender_address`, `exception_recipients` to be source from configurable parameters as mentioned in the napkin note.